index.js

```diff
@@ -79,26 +81,16 @@ cli.on('loader:success', function(name) {
   // However, we don't want to show the mjs-stub loader in the logs
   /* istanbul ignore else */
   if (path.basename(name, '.js') !== 'mjs-stub') {
-    log.info('Loaded external module:', chalk.magenta(name));
+    log.info(msgs.info.loaderSuccess, name);
   }
 });
 
 cli.on('loader:failure', function(name, error) {
-  log.warn(
-    chalk.yellow('Failed to load external module:'),
-    chalk.magenta(name)
-  );
-  /* istanbul ignore else */
-  if (error) {
-    log.warn(chalk.yellow(error.toString()));
-  }
+  log.warn(msgs.warn.loaderFailure, name, Boolean(error), error.toString());
 });
 
 cli.on('respawn', function(flags, child) {
-  var nodeFlags = chalk.magenta(flags.join(', '));
-  var pid = chalk.magenta(child.pid);
-  log.info('Node flags detected:', nodeFlags);
-  log.info('Respawned to PID:', pid);
+  log.info(msgs.info.respawn, flags.join(', '), child.pid);
```

Comment:
These should not be combined, one message is about detecting node flags and one if about the respawn.

Reply:
It seems that users configuring this would consider these two lines as one.
Additionally, in this way, it is also possible to either output only one line or not output both lines.